docs/i18n.rst

```diff
@@ -119,10 +104,8 @@ Python code, HTML templates, or desktop icon labels, the translation
 files should also be updated by running ``make translate`` in the root
 of the SecureDrop working copy.
 
-The ``translate`` target runs ``i18n_tool.py translate-messages`` and
-``i18n_tool.py translate-desktop``, which in turn use ``pybabel
-extract`` to gather source strings.  These commands update the `.pot`_
-files for the SecureDrop server code and the desktop icons, as well as
+The ``translate`` target gathers source strings, then updates the 
+`.pot`_ files for the SecureDrop server code and the desktop icons, as well as
 the `.po`_ files for each language.
```

Comment:
```suggestion
The ``extract-strings`` target gathers source strings, then updates the 
`.pot`_ files for the SecureDrop server code and the desktop icons.
```

The new approach defers the `.pot`-to-`.po` merge to Weblate.  See freedomofpress/securedrop-client#1657 for a tidy example.  This would also remove the paragraph beginning "Then review...".

Reply:
That makes sense, thanks for clarifying!